build.info

```diff
@@ -75,6 +75,9 @@ GENERATE[include/openssl/x509_vfy.h]=include/openssl/x509_vfy.h.in
 GENERATE[include/crypto/bn_conf.h]=include/crypto/bn_conf.h.in
 GENERATE[include/crypto/dso_conf.h]=include/crypto/dso_conf.h.in
```

Comment:
There are a pile more here too ;)
They can wait - i.e. not for this PR.

Reply:
> There are a pile more here too ;)

Not really.  All of those you see are harmless, in the sense that the base name itself *is* the perl package name for them, or they don't have a package name at all.  I suspect that the main example you have is the use of `doc/perlvar.pm`.  Of course, you *can* specify its use as `DEPEND[whatever.pod]=..|perlvar.pm` just for the sake of it, but that makes no difference, as that's the assumption when nothing else is said.

Reply:
Ah, didn't realise.

Having a dependency on the generation script is probably a good idea nonetheless.

Reply:
Yeah well, it's there, so...